public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -153,9 +165,20 @@ class UnConnectedExploreToolbar extends PureComponent<Props> {
                 Split
               </ToolbarButton>
             ) : (
-              <ToolbarButton tooltip="Close split pane" onClick={() => closeSplit(exploreId)} icon="times">
-                Close
-              </ToolbarButton>
+              <>
+                <ToolbarButton
+                  title={`${isLargerExploreId ? 'Narrow' : 'Widen'} pane`}
+                  disabled={isLive}
+                  onClick={onClickResize}
+                >
+                  {(exploreId === 'left' && isLargerExploreId) || (exploreId === 'right' && !isLargerExploreId)
+                    ? '<'
+                    : '>'}
+                </ToolbarButton>
```

Comment:
```suggestion
                <ToolbarButton
                  tooltip={`${isLargerExploreId ? 'Narrow' : 'Widen'} pane`}
                  disabled={isLive}
                  onClick={onClickResize}
                  icon={
                    (exploreId === 'left' && isLargerExploreId) || (exploreId === 'right' && !isLargerExploreId)
                      ? 'angle-left'
                      : 'angle-right'
                  }
                />
```

we should probably use icons instead of text for the button, also, afaik, there has been some talk around stopping using titles for buttons in favor of tooltips, which internally should also provide a label to the button

![Screenshot 2022-09-20 at 11 41 00](https://user-images.githubusercontent.com/1170767/191237377-210d1e7a-6ef3-417b-acee-9e4f5dda7792.png)